content/kubermatic/master/guides/addons/aws_node_termination_handler/_index.en.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "AWS Node Termination Handler"
+date = 2021-01-27T14:07:00+02:00
+weight = 6
+
++++
+
+## AWS Spot Instance
+
+Kubermatic Kubernetes Platform(KKP) is capable of creating different machine types in more than ten different cloud providers.
+One of those providers is AWS. 
+
+AWS cloud provider supports a special machine instances that are called spot instances. Spot instances(according to aws official 
+documentations) are instances that use spare EC2 capacity that is available for less than the On-Demand price.
+Because Spot Instances enable you to request unused EC2 instances at steep discounts, you can lower your Amazon EC2 costs significantly.
+The hourly price for a Spot Instance is called a Spot price.
+
+Kubermatic Kubernetes Platform supports the creation of aws spot instances and use them as worker nodes for any aws user cluster.
+Spot Instances are very good choice in case of cost-effectiveness if you can be flexible about when your applications run and if 
+your applications can be interrupted.
+
```

Comment:
Does it make sense to add a warning here that you shouldn't use AWS Spot instances for the stateful workload?

Reply:
Good point! will add it